clang/lib/Frontend/CompileJobCache.cpp

```diff
@@ -724,7 +724,7 @@ Error ObjectStoreCachingOutputs::finishComputedResult(
 std::optional<int> ObjectStoreCachingOutputs::replayCachedResult(
     const llvm::cas::CASID &ResultCacheKey, llvm::cas::ObjectRef ResultID,
     bool JustComputedResult) {
-  if (JustComputedResult && !ComputedJobNeedsReplay)
+  if (JustComputedResult)
```

Comment:
I doubt this works. This will probably cause the output being written twice when CASBackend or CASID output is used.